starlite/route_map.pyi

```diff
@@ -1,43 +1,36 @@
-# A proxy for a Rust extension class that implements the interface below
-# The Rust implementation is located at ./starlite/extensions/rust
```

Comment:
Maybe this comment should be added back (with the correct path of `./extensions/rust`) as an indicator to other people reading the code not aware that this file is type hinting for a Rust extension